sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_automl_dsl_pipeline.py

```diff
@@ -105,9 +106,10 @@ def train_automl_regression_in_pipeline(regression_train_data):
         )
 
         pipeline_job: PipelineJob = train_automl_regression_in_pipeline(regression_train)
+        pipeline_job.settings.default_compute = "cpu-cluster"
 
         from_rest_pipeline_job = client.jobs.create_or_update(pipeline_job)
-        assert_final_job_status(from_rest_pipeline_job, client, PipelineJob, JobStatus.COMPLETED)
+        cancel_job(client, from_rest_pipeline_job)
```

Comment:
how long does it take to run these automl tests in live mode?

Reply:
I have tested locally on canary region, it takes around 7.5 minutes to run all these tests
![image](https://user-images.githubusercontent.com/38847871/201566613-3da6e960-1deb-47ef-834b-bf5a7c3ef24d.png)